### ISSUE TITLE
Support publishing surveys privately

### ISSUE THREAD

**Issue by MattF-NSIDC** [nsidc/usaon-benefit-tool#120]
* Admins check a box to make it private
* When an assessment is private, only admins can see it

**Comment by rmarow** [comment]
Current state: have boolean field  default surveys to not private for now.

**Comment by MattF-NSIDC** [comment]
Interesting catch; should `private` actually be a state, or just an attribute on survey? I prefer having them as an attribute on the survey, and when their state is `published`, they only show to people with permissions. This makes the `state` flowchart simpler, which is always a win :)

**Comment by hazelshapiro** [comment]
Approved 'private' being an attribute

**Comment by mfisher87** [comment]
@hazelshapiro with `private` as an attribute, what should the user experience be around that?

For example:

* Admins can check a box to set an assessment to private
* When an assessment is private, only admins can see it

Or does this depend on a story allowing assigning respondents to surveys, because we want private surveys to be visible to only respondents?

**Comment by mfisher87** [comment]
We decided:

* Admins check a box to make it private
* When an assessment is private, only admins can see it

After #66 , Respondents will also be able to see their assigned private surveys, but that's not part of this ticket.
